src/ape/_utils.py

```diff
@@ -0,0 +1,35 @@
+from typing import Dict, Iterator, List, Union
+
```

Comment:
Wait, so we have `utils` and `_utils`? Not sure I like that

Reply:
I want a way to make shared utils that are non-public

Reply:
I wonder if this could live under `ape/contracts/_utils.py`